### DISCUSSION TITLE
List residency attendees on the site

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#520] [Ideas]
Talking with @lkavanagh30 the other day we were thinking about the amazing people who come to the residencies ... and wouldn't it be nice to be able to show them on the site (having checked they were happy to be listed publicly of course!)

I was imaging this could be a nice simple "table" of faces with short info / bio.

**Comment by rufuspollock** [comment]
@nathenf @Postnormalval @msantolini @laurenwigmore @khalilcodes any thoughts on this one e.g in terms of ...

- Desirability
- Features
- Implementation (@khalilcodes)

**Comment by msantolini** [comment]
I think that's a great idea - similar to what they did at [Fey](https://feytopia.com/en/community) or at [PAF](https://www.pa-f.net/) (for upcoming visitors). Definitely worth it for people to get a feel of their potential fit. One potential features of interest might be "residencies attended" maybe.

**Comment by khalilcodes** [comment]
@rufuspollock In terms of implementation, I'm thinking we have the required info in for eg. `people/residencies/...` and can be displayed at a dedicated page maybe `/community`.

**Comment by laurenwigmore** [comment]
@rufuspollock I agree, I think it would be a really nice addition and it could help new people to navigate whether a residency is for them or not, and also support community building. Plus people are curious generally! 
Do you have a past residency participants at github forum? (I couldn't see one, but maybe there are too many already or one functioning like this). This could be set up as well as a way to keep people specifically from the residences connected, although I'm not sure how well it would be used, but it could also be a way to promote up and coming or follow on residencies as well.

**Reply by rufuspollock** [reply]
> Do you have a past residency participants at github forum? (I couldn't see one, but maybe there are too many already or one functioning like this). 

To answer this one specific question: No we don't atm.
